### ISSUE TITLE
Mockup Design Updates

### ISSUE THREAD

**Issue by selinisik** [bounswe/bounswe2023group4#201]
### Problem Definition

We decided to update our mockup design considering more UI/UX design principles. Creating a detailed design will make the implementation process of frontend and mobile easier.

**Comment by selinisik** [comment]
We have discussed the new design with @enfurars and @kutaysaran then created the web view of the mockup. You can see updated desktop version [here](https://github.com/bounswe/bounswe2023group4/wiki/Mock-up).

**Comment by enfurars** [comment]
We have decided to discuss the new version of mobile mock-up further as new information about **"how mobile experience should and should not be like"** is given in the discord channel. So as it can be seen from the link provided by @selinisik we did not uploaded updated version of mobile mock-ups. We will have a new issue considering our mobile users' experience on the application and apply our new design with a mobile friendly interface.
